docs/reference/api/drivers/RawSerial.md

```diff
@@ -1 +1,34 @@
 ## RawSerial
+
+The RawSerial interface provides UART functionality without the use of streams like the Serial class. This makes it suitable for use in interrupt handlers with the RTOS.
+
+Serial channels have the following configurable parameters:
+
+  - _Baud Rate_ - Standard baud rates range from a few hundred bits per second to megabits per second. The default setting for a serial connection on the Mbed microcontroller is 9600 baud.
+  - _Data length_ - Transferred data can be either 7 or 8 bits long. The default setting for a serial connection on the Mbed microcontroller is 8 bits.
+  - _Parity_ - You can add an optional parity bit. The parity bit will be automatically set to make the number of 1s in the data either odd or even. Parity settings are *Odd*, *Even* or *None*. The default setting for a serial connection on the Arm Mbed microcontroller is None.
+  - _Stop Bits_ - After data and parity bits have been transmitted, one or two stop bits are inserted to "frame" the data. The default setting for a serial connection on the Mbed microcontroller is one stop bit.
```

Comment:
Query: Who or what is transmitting the data and parity bits? Also, who or what is inserting the stop bits to "frame" the data? Please clarify for active voice.

Reply:
These are configurable settings for the RawSerial object, the transmitting is being done by that object on calls to write/putc. 

Would something like 
>After data and parity bits have been transmitted, the RawSerial object inserts one or two stop bits to "frame" the data.

be preferable?

Reply:
Yes. 
Also, do we know who or what is transmitting the data and parity bits? If not, maybe we could do "After the transmission of data and parity bits, the RawSerial object inserts one or two stop bits to "frame" the data."?

Reply:
In parity, please change it to "The object automatically sets the parity bit to make the number of..."